### ISSUE TITLE
Mapping issue

### ISSUE THREAD

**Issue by riemers** [Norien/Home-Assistant-Config#202]
https://github.com/Norien/Home-Assistant-Config/blob/f2affdaa0d0d5a056c8c79a62b0196fa343a7836/packages/vacuum.yaml#L120

Whenever HA restart i get the message:

```
Could not render template Status: UndefinedError: 'mappingproxy object' has no attribute 'battery_level'
```

I think above referenced one can be replaced with: 
```
value_template: "{{ state_attr('vacuum.roomba','battery_level')|int }}"
```

Only below that is another battery_level and that trick doesn't work. Maybe its something simple but i don't see it 1.2.3. ;p

Any thoughts?

**Comment by Norien** [comment]
Hi, I want to look into this but my Roomba died and I have just ordered a replacement this week. Hopefully it arrives next week but it's a different model and i'm not 100% sure I'll be able to get into HASS. 

I actually removed that sensor a while back as breaking out the battery is no longer of use to me since I now use a custom lovelace card to auto grab batteries and display them!

I should really set some time to once again update my git to current code.

thanks for reaching out!
